common/interfaces/desktop_app_interface/usb_api.c

```diff
@@ -208,32 +211,36 @@ void usb_send_byte(const uint32_t command_type, const uint8_t byte) {
 void usb_send_data(const uint32_t command_type,
                    const uint8_t *transmit_data,
                    const uint32_t size) {
-  uint16_t proto_len = 0;
-  uint8_t offset = 0;
+  return;
+}
+
+void usb_send_msg(const uint8_t *msg, const uint32_t size) {
+  uint8_t len_size = 4;
```

Comment:
Validate pointers and size variables input arguments

Reply:
added